aws-js-s3-folder/README.md

```diff
@@ -3,50 +3,29 @@
 A static website that uses [S3's website support](https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteHosting.html).
 For a detailed walkthrough of this example, see the tutorial [Static Website on AWS S3](https://pulumi.io/quickstart/aws-s3-website.html).
 
-## Deploying and running the program
+## Getting Started (Pulumi Cloud Console)
 
-Note: some values in this example will be different from run to run.  These values are indicated
-with `***`.
+Create a new project from this example in the [Pulumi Cloud Console](https://app.pulumi.com/) by clicking the 'Deploy with Pulumi' button:
 
-1.  Create a new stack:
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-    ```bash
-    $ pulumi stack init website-testing
-    ```
+After creating the project, a command will be provided (e.g. `pulumi new https://github.com/pulumi/examples/aws-js-s3-folder -s <owner>/<stack-name>`) that you can run to get the source code and preview an initial deployment.
 
-1.  Set the AWS region:
+When you run the command, you'll be presented a preview of the AWS resources that will be created.  Once you've reviewed and selected "Yes", the deployment will begin.
 
-    ```
-    $ pulumi config set aws:region us-west-2
-    ```
+## Getting Started (Command Line)
```

Comment:
I'm thinking about cutting this section. On the one hand, it's nice to show the single command that you can run to get started. On the other hand, it just adds more words to the README, making it seemingly more complex, when showing just the button would be sufficient. Thoughts?